r/R/dplyr-functions.R

```diff
@@ -108,6 +108,10 @@ nse_funcs$is.infinite <- function(x) {
   is_inf & !nse_funcs$is.na(is_inf)
 }
 
+nse_funcs$abs <- function(x){
```

Comment:
A super nit-picky whitespace suggestion:

```suggestion
nse_funcs$abs <- function(x) {
```